docs/source/api.rst

```diff
@@ -64,14 +85,19 @@ See also :ref:`threading` for more information on multithreading in UDFs.
 .. versionadded:: 0.9.0
 
 The :class:`~libertem.executor.concurrent.ConcurrentJobExecutor` runs all tasks
-using :mod:`python:concurrent.futures`. Using the
-:class:`python:concurrent.futures.ThreadPoolExecutor` allows sharing large
-amounts of data as well as other resources between main thread and workers
-efficiently, but is severely slowed down by the Python `global interpreter lock
-<https://wiki.python.org/moin/GlobalInterpreterLock>`_ under many circumstances.
-Furthermore, it can create thread safety issues such as
+using :mod:`python:concurrent.futures`. Using a
+:class:`python:concurrent.futures.ThreadPoolExecutor`, which is the deafult behaviour,
+allows sharing large amounts of data as well as other resources between the
+main thread and workers efficiently, but is severely slowed down by the
+Python `global interpreter lock <https://wiki.python.org/moin/GlobalInterpreterLock>`_
+under many circumstances. Furthermore, it can create thread safety issues such as
 https://github.com/LiberTEM/LiberTEM-blobfinder/issues/35.
 
+It is also in principle possible to use a :class:`python:concurrent.futures.ProcessPoolExecutor`
+as backing for the :class:`~libertem.executor.concurrent.ConcurrentJobExecutor`,
+though this is untested and is likely to lead to worse performance than the
+LiberTEM default :class:`~libertem.executor.dask.DaskJobExecutor`.
+
```

Comment:
I tried that, actually, and it didn't work. I don't remember how exactly it failed, though. Maybe one could add an xfail test case just for completeness sake, since we both had the same idea independently? :-)

Reply:
It did run for me, but only when I wasn't pickling a local function / closure, specifically.

Reply:
Ah right, that was it!